stripe/src/main/java/com/stripe/android/model/BankAccount.java

```diff
@@ -44,8 +44,89 @@
     @Nullable private String mLast4;
     @Nullable private String mRoutingNumber;
 
+    public static Builder newBuilder(
+            @NonNull String accountNumber,
+            @Nullable String routingNumber,
+            @NonNull @Size(2) String countryCode,
+            @NonNull @Size(3) String currency) {
+        return new Builder(accountNumber, routingNumber, countryCode, currency);
+    }
+
+    public static final class Builder {
```

Comment:
Looking at `Card.Builder`, I see class documentation and docs for some of the public methods (constructor, `build()`).

Should you add docs, or is it pretty self-evident what it's for?